packages/server/engine-lib/engLib/task/headers/task.hpp

```
@@ -118,6 +118,16 @@ class ITask {
     virtual Error execute() noexcept;
     static Text buildType(json::Value &cmd) noexcept;
 
+    //-----------------------------------------------------------------
+    /// @details
+    ///		The complete task-file JSON of the task currently executing
+    ///		in this process — null when no task is running. Published
+    ///		for the python binding rocketlib.getTask() so subprocess
+    ///		python (e.g. the storage layer reading identity and the
+    ///		storage anchor) needs no per-endpoint plumbing.
+    //-----------------------------------------------------------------
+    static json::Value currentTask() noexcept;
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)